extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -191,19 +195,23 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 		}
 	}
 
-	function refreshSlot(slot) {
-		log(['Refresh slot', slot.name, slot], log.levels.debug, logGroup);
-		refreshTargetingData(slot);
-		googleTag.refreshSlot(slot);
+	function refreshSlot(slotName) {
+		var slot = googleSlots.getSlotByName(slotName),
+			slotElement;
+
+		if (slot) {
+			log(['Refresh slot', slotName, slot], log.levels.debug, logGroup);
+			slotElement = doc.getElementById(slot.getSlotElementId());
+			refreshTargetingData(slot);
+			googleTag.refreshSlot(slot);
+			slotElement.setAttribute('data-gpt-slot-params', JSON.stringify(gptTargeting.getSlotLevelTargeting(slotName)));
```

Comment:
We already perform similar operation in https://github.com/Wikia/app/blob/dc997090f63f18e6ee7e6a1c6ee52cd0d6aed2ff/extensions/wikia/AdEngine/js/provider/gpt/adElement.js#L83-L83

Can we put this logic in one place?

Reply:
done